lib/demangler.js

```diff
@@ -60,6 +60,38 @@ class Demangler extends AsmRegex {
         this.quadUnderscoreDef = /\.quad\s*(_[a-z0-9$_@.]*)/i;
     }
 
+    // Iterates over the labels, demangle the label names and updates the start and
```

Comment:
Wow! I hadn't thought of this. Thank you for your hard work in discovering this, and fixing this too. I wonder if this can be done more efficiently? seems like there's going to be `O(n*m)` iterations here, for each `n` translations, we loop over all `m` labels. Perhaps passing a map of `{old: new}` would help? Not necessary for this PR (perhaps a later PR or similar?)

Reply:
The number of labels in a line is really small. Most of the time it is only 1 or 2. But you are right, we can think on a better solution for this :)

Reply:
Very true! This is totally fine for now...I'm only sensitive as we have a few bugs outstanding where the site hangs for a bit on the server.